sensing/pointcloud_preprocessor/launch/ring_passthrough_filter.launch.xml

```diff
@@ -1,25 +1,26 @@
 <launch>
   <arg name="input_topic_name" default="/sensing/lidar/top/pointcloud_raw"/>
   <arg name="output_topic_name" default="/sensing/lidar/top/ring_passthrough_filtered/pointcloud"/>
-
   <arg name="filter_field_name" default="ring"/>
```

Comment:
With the parameter file, you could remove some of the parameters in this file just like what you did in other launch file.